src/amo/components/RatingManager/index.js

```diff
@@ -97,7 +97,7 @@ export class RatingManagerBase extends React.Component {
         return i18n.gettext('Log in to rate this dictionary');
       case ADDON_TYPE_LANG:
         return i18n.gettext('Log in to rate this language pack');
-      case ADDON_TYPE_SEARCH:
+      case ADDON_TYPE_OPENSEARCH:
         return i18n.gettext('Log in to rate this search engine');
```

Comment:
Yes actually! "search plugin" I think works (because it's not always searching via a search engine).